File: Meta/META-INF/resources/lark/org/bark/PersonBean.js
(function(){ 
  function PersonBean() {    
  }
  PersonBean.prototype.__proto__ = Object.prototype;
  __cache["org.bark.PersonBean"] = PersonBean;
  PersonBean.prototype.__readObject = function(json, handlers, obj) {};
  PersonBean.prototype.__writeObject = function(obj, handlers) {};
  PersonBean.prototype.__class = new (__lc('java.lang.Class'))("org.bark.PersonBean", PersonBean, Object.prototype.__class, [], 1);
  return  PersonBean;
})();

File: Meta/META-INF/resources/lark/org/bark/PersonBean.java
package org.bark;

@RemotingBean
public class PersonBean {
  static {
    org.bark.remoting.SerializerFactory.getInstance().register(PersonBean.class, new org.bark.remoting.Serializer(){
      @Override
      public void writeObject(javax.json.JsonObjectBuilder builder, org.bark.remoting.ReferenceProcessor handler, Object value) {
      }
    });
    org.bark.remoting.DeserializerFactory.getInstance().register(PersonBean.class, new org.bark.remoting.Deserializer(){
      @SuppressWarnings({ "unchecked", "rawtypes" })
      @Override
      public Object readObject(javax.json.JsonObject jsonObj, Object[] handlers, Object obj) throws Exception {
      }
    });
  }
}